javascript/javascript1/week1/exercises/exercises.js

```diff
@@ -0,0 +1,54 @@
+// var cross = 24 * 55;
+// console.log(cross);
+
```

Comment:
Remember to clean up your code 🧹

Reply:
I added they for lesson backup. Next time i will dont add it. Its looking i was make a mistake. Sorry for that.